### ISSUE TITLE
Add some comparation benchmarks 

### ISSUE THREAD

**Issue by danielmeza** [KirillOsenkov/XmlParser#36]
Add benchmarks to make a comparation with other libraries

**Comment by KirillOsenkov** [comment]
Good idea, I've always wondered how it compares to https://github.com/mhutch/MonoDevelop.Xml for example.
